software/caddy/en.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "caddy"
+lastmod = "2021-02-01T15:20:50-03:00"
++++
+# caddy
+
+Solus provides the caddy web server and proxy to users and developers who want to test their web applications before setting them to production.
+
+Remember that Solus is not a server operating system: any usage outside local testing is discouraged.
+
+## Installation
+
+caddy is available in the `caddy` package. It can be installed from the [Software Center](https://getsol.us/articles/common-tasks/installing-and-updating-software/en/) or via terminal:
```

Comment:
URL should be relative e.g. `[Software Center](/articles/common-tasks/installing-and-updating-software/en/)`